syntax = "proto3";

package protobuff;

message EntityPb {
    string id = 1;
    StatusEnum lifeTime = 2;
}

enum StatusEnum {
    UNKNOWN = 0;
    ACTIVE = 1;
    DELETED = 2;
    SUSPUNDED = 3;
    BLOCKED = 4;
}
